### PR TITLE
update makefile

### PR DESCRIPTION
**Pull request by nkubala**
Brings the makefile more in line with the other makefiles in our repositories.

cc @dlorenc @r2d4

### CODE REVIEW

File: structure_tests/Makefile [hunk 1]
```diff
@@ -19,22 +19,41 @@ VERSION_BUILD ?= 0
...
+	GOOS=$* GOARCH=$(GOARCH) CGO_ENABLED=0 go test -c . -o $@ $(BUILD_PACKAGE)
 
-.PHONY: release
-release: structure-test
-	gsutil cp $(BUILD_DIR)/structure-test $(UPLOAD_LOCATION)/$(VERSION)/structure-test-amd64
+%.sha256: %
+	shasum -a 256 $< &> $@
+
+%.exe: %
+	mv $< $@
```
Comment (r2d4) [inline structure_tests/Makefile]: i think this is an error in container-diff too, but this should probably be a `cp` instead of a `mv`

Comment (dlorenc) [reply structure_tests/Makefile]: Yeah, should be cp. I think @r2d4 just fixed this somewhere, maybe in minikube?

File: structure_tests/Makefile [hunk 2]
```diff
@@ -19,22 +19,41 @@ VERSION_BUILD ?= 0
 
 VERSION ?= v$(VERSION_MAJOR).$(VERSION_MINOR).$(VERSION_BUILD)
 
+GOOS ?= $(shell go env GOOS)
+GOARCH = amd64
+ORG := github.com/GoogleCloudPlatform
+PROJECT := runtimes-common/structure_tests
+REPOPATH ?= $(ORG)/$(PROJECT)
+RELEASE_BUCKET ?= $(PROJECT)
```
Comment (dlorenc) [inline structure_tests/Makefile]: Don't we have another bucket for some of our other tools? It would be nice to consolidate these.

Comment (nkubala) [reply structure_tests/Makefile]: ah whoops meant to change this. I don't think we have a consolidated one though, but I can make one

**Comment by nkubala** [comment]
I signed it!

File: structure_tests/Makefile [hunk 3]
```diff
@@ -19,22 +19,38 @@ VERSION_BUILD ?= 0
 
 VERSION ?= v$(VERSION_MAJOR).$(VERSION_MINOR).$(VERSION_BUILD)
 
+GOOS ?= $(shell go env GOOS)
+GOARCH = amd64
+PROJECT := runtimes-common/structure_tests
+RELEASE_BUCKET ?= gcp-container-tools/structure-test
+
+SUPPORTED_PLATFORMS := linux-$(GOARCH) darwin-$(GOARCH) windows-$(GOARCH).exe
```
Comment (r2d4) [inline structure_tests/Makefile]: does this actually build with darwin and windows?

Comment (nkubala) [reply structure_tests/Makefile]: it builds on linux/darwin, but there's an issue with windows currently. planning on fixing in a follow-up PR

Comment (r2d4) [reply structure_tests/Makefile]: i would just remove the windows entry then, since cross build will fail right?

**Pull request merged**